idaes/models_extra/power_generation/costing/power_plant_capcost.py

```diff
@@ -0,0 +1,1995 @@
+                    sum(self.variable_operating_costs[0, i] for i in non_fuel_resources)
+                )
+                / self.capacity_factor
+                / 12
+            )
+
+            if land_cost is not None:
+                if (
+                    (type(land_cost) in [Expression, ScalarExpression])
+                    or (type(land_cost) is Param and land_cost.get_units is None)
+                    or (type(land_cost) is Var and land_cost.get_units is None)
+                ):
+                    land_cost_units_factor = 1 * CE_index_units
+                else:
+                    land_cost_units_factor = 1
+
+            self.total_overnight_capital = Expression(
+                expr=self.total_TPC
+                # pre production costs
+                + self.six_month_labor
+                + (
+                    self.maintenance_material_cost / 12  # 1 month maintenance materials
+                    + self.non_fuel_and_waste_OC  # 1 month non fuel consumables
+                    + (
+                        self.waste_costs_OC if waste is not None else 0 * CE_index_units
+                    )  # 1 month waste
+                    # inventory capital costs
+                    + (
+                        self.fuel_cost_OC if fuel is not None else 0 * CE_index_units
+                    )  # 60 day fuel supply
+                    # Other costs
+                    + (
+                        self.chemical_costs_OC
+                        if waste is not None
+                        else 0 * CE_index_units
+                    )  # Initial Cost for Catalyst and Chemicals
+                )
+                * 1
+                * pyunits.year  # variable costs for 1 year
+                + (
+                    land_cost * land_cost_units_factor
+                    if land_cost is not None
+                    else 0 * CE_index_units
+                )
+                + self.total_TPC
+                * self.pct_TPC  # other owners costs (other + spare parts + financing + other pre-production)
+            )
+
+            self.tasc_toc_factor = Param(
+                initialize=1.093,
+                mutable=True,
+                doc="TASC/TOC factor from Exhibit 3-7 reference 1, real for three years 1.093, real for five years 1.154, nominal for three years 1.242, nominal for five years 1.289",
+            )
+
+            self.total_as_spent_cost = Expression(
+                expr=self.total_overnight_capital * self.tasc_toc_factor
+            )
+
+            self.fixed_charge_factor = Param(
+                initialize=0.0707,
+                mutable=True,
+                doc="Fixed charge rate from Exhibit 3-5 based on CRF values, real for three/five years 0.0707, nominal for three/five years 0.0886",
+            )
+            self.annualized_cost = Expression(
+                expr=self.fixed_charge_factor * self.total_as_spent_cost
+            )
+
+            self.add_COE_cost = Param(
```

Comment:
Maybe using a more descriptive name for it to keep consistence, for example "additional_cost_of_electricity"